chall07/llahti.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+#Python 3.8.2
+import sys
+import math
+
+def snail(lines):
+    size = len(lines[0])
+    i = 0
+    while (i < size / 2):
+        if size % 2 == 1 and i == math.floor(size / 2):
```

Comment:
why do you need math.floor() here? does int(size/2) wouldn't be enough?

I'm not sure